src/Analysis/Engine/Test/FindReferencesTests.cs

```diff
@@ -878,6 +885,9 @@ class bcd(abc):
             var fobUri = TestData.GetTempPathUri("fob.py");
             var oarUri = TestData.GetTempPathUri("oar.py");
             using (var server = await CreateServerAsync(PythonVersions.LatestAvailable3X)) {
+                await server.LoadFileAsync(fobUri);
```

Comment:
`Server.DidOpenTextDocument` calls the same `AddFileAsync` as `LoadFileAsync` (`ProjectFiles.GetEntry` is a simple dictionary lookup):
https://github.com/Microsoft/python-language-server/blob/39a17b1a8aefed9ee18d904ceff2d3c447f0f060/src/LanguageServer/Impl/Implementation/Server.cs#L192

So if these loads change anything in test results, it is not because of the codeflow, but because of the timings.

Reply:
Or, maybe calling `EnqueueItem` twice really changes something, but that isn't expected either.

Reply:
There is known problem with files being analyzed while other files are still being added. I am going to get my `AnalysisQueue.Start/Stop` change and see if that works better.

Reply:
AnalysisQueue.Start/Stop works OK, no need to double load/open